<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我爱地大之校园导航</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
border - radius:25px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="qview" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string comment="转换图" extracomment="转换图"/>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>卫星</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>始发地</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>体育馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教一楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教二楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教三楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>#52#53栋</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>公主楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生一食堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生三食堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>信息楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>信工楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图书馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新峰公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>博物馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鸿毅堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大学生活动中心</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主楼</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>体育馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教一楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教二楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教三楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>#52#53栋</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>公主楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生一食堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生三食堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>信息楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>信工楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图书馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新峰公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>博物馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鸿毅堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大学生活动中心</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主楼</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>开始搜索</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>终点站</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>清除路线</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>关闭音乐</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>播放音乐</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qview</class>
   <extends>QGraphicsView</extends>
   <header location="global">qview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
